plugins/ingest-geoip/src/main/java/org/elasticsearch/ingest/geoip/GeoIpProcessor.java

```diff
@@ -185,6 +185,14 @@ DatabaseReader getDbReader() {
                         geoData.put("continent_name", continentName);
                     }
                     break;
+                case REGION_ISO_CODE:
+                    String countryIso = country.getIsoCode();
+                    String subdivisionIso = subdivision.getIsoCode();
+                    if (countryIso != null && subdivisionIso != null) {
+                        String regionIsoCode = countryIso + "-" + subdivisionIso;
```

Comment:
Instead of combining these and creating our own unique "region" iso code (which I don't think is actually a thing?) can we expose the subdivision by itself? The rest of the geoip data is exposed directly from the database without alteration or manipulation, and it would be nice to keep that simplicity.

Reply:
The concatenated code is a real ISO code. For example, [here are the Canadian provinces](https://www.iso.org/obp/ui/#iso:code:3166:CA). For some reason the subdivision code exposed by the GeoLite database nixes the country code portion of the full code (AB for Alberta, Canada instead of CA-AB). 

Including the full region ISO code would be imperative to inner join to boundary layers in Kibana region maps. This is because the full ISO code (ex. 'IE-C' for Connacht, Ireland, 'ES-C' for A Coruña, Spain) is distinct between countries whereas the portion exposed by the GeoLite database (ex. 'C') is not.

Reply:
@nickpeihl agreed, the concatenation is a real code and uniquely identifies an administrative region within a country. It would be very useful to have this exposed.

Reply:
Thanks for the info, I see the specification now. Can you please add a comment that this is an ISO 3166-2 code?